src/PostSharp.Engineering.BuildTools/Build/Model/Product.cs

```diff
@@ -1367,9 +1383,9 @@ public bool BumpVersion( BuildContext context, BuildSettings settings )
             // When bumping locally, the product with MainVersionDependency is not allowed to be manually bumped.
             if ( this.MainVersionDependency != null )
             {
-                context.Console.WriteWarning( "Version of a product derived from MainVersionDependency cannot be bumped." );
+                context.Console.WriteError( "Version of a product derived from MainVersionDependency cannot be bumped." );
```

Comment:
```suggestion
                context.Console.WriteError( $"The version would need to be bumped, but it cannot because the MainVersion is dependent on {this.MainVersionDependency}. Create a fake change in  {this.MainVersionDependency} and bump this repo." );
```